portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,13 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.center {
+  display: block;
+  margin: 0 auto; 
+}
+
+button.center {
```

Comment:
Doesn't seem like this (lines 16-19) should be necessary given that i defined the same 'center' class above but it doesn't work on the button without this class.

Reply:
I think it might've been messed up because I nested a button inside of the anchor instead of just making the anchor a button-- changing that and setting text-align to center works